### DISCUSSION TITLE
Drag and drop reordering layer

### DISCUSSION THREAD

**Discussion by rgstd** [Orama-Interactive/Pixelorama#849] [Ideas]
Maybe your great application will more interactive if drag and drop reorder layer between two layers exist.
This app can reorder layer by just drag n drop to another layer, but it is not placing layer below or above layer, but replacing targeted layer.

Windows 11
Pixelorama v0.10.3-stable non-portable version

**Comment by Variable-ind** [comment]
This feature has been implemented and will be available in next version :)
If you want you could also try the nightly build
https://nightly.link/Orama-Interactive/Pixelorama/workflows/dev-desktop-builds/master
(though it may be unstable and not recommended for production)

**Reply by rgstd** [reply]
Thank you so much. Sorry for the bother. Hehe.

**Reply by Variable-ind** [reply]
No worries. Any suggestion is welcome 😁
